data/json/mapgen/house12.json

```diff
@@ -0,0 +1,129 @@
+                { "point": "terrain", "id": "t_dirt", "x": [ 2, 5 ], "y": [ 0, 7 ], "repeat": [ 8, 11 ] },
+                { "point": "terrain", "id": "t_dirt", "x": [ 2, 23 ], "y": [ 21, 23 ], "repeat": [ 8, 14 ] },
+                { "point": "terrain", "id": "t_dirt", "x": [ 3, 4 ], "y": [ 9, 10 ], "repeat": [ 0, 1 ] },
+                { "point": "terrain", "id": "t_dirt", "x": [ 8, 11 ], "y": [ 0, 7 ], "repeat": [ 8, 11 ] },
+                { "point": "terrain", "id": "t_dirt", "x": 9, "y": [ 9, 10 ], "repeat": [ 0, 1 ] },
+                { "point": "terrain", "id": "t_dirt", "x": [ 10, 11 ], "y": 9, "repeat": [ 0, 1 ] },
+                { "point": "terrain", "id": "t_dirt", "x": [ 12, 23 ], "y": [ 0, 4 ], "repeat": [ 8, 14 ] },
+                { "point": "terrain", "id": "t_dirt", "x": 14, "y": [ 14, 15 ], "repeat": [ 0, 1 ] },
+                { "point": "terrain", "id": "t_dirt", "x": [ 15, 23 ], "y": 20, "repeat": [ 1, 2 ] },
+                { "point": "terrain", "id": "t_dirt", "x": [ 20, 23 ], "y": [ 14, 19 ], "repeat": [ 5, 8 ] },
+                { "point": "terrain", "id": "t_dirt", "x": [ 21, 23 ], "y": [ 5, 13 ], "repeat": [ 6, 9 ] }
+            ],
+            "place_groups": [
+                { "item": "dresser", "chance": 70, "x": 19,  "y": 6 },
+                { "item": "dresser", "chance": 70, "x": 13,  "y": 9 },
+                { "item": "homebooks", "chance": 100, "x": 13,  "y": 6 },
+                { "item": "dresser", "chance": 80, "x": 13,  "y": 19 },
+                { "item": "office", "chance": 20, "x": [ 18, 19 ],  "y": 12 },
+                { "item": "dresser", "chance": 70, "x": 18,  "y": 14 },
+                { "item": "dresser", "chance": 70, "x": 16,  "y": 14 },
+                { "item": "softdrugs", "chance": 75, "x": [ 16, 18 ],  "y": [ 16, 17 ] },
+                { "item": "fridge", "chance": 50, "x": 11,  "y": 15 },
+                { "item": "oven", "chance": 50, "x": 11,  "y": 17 },
+                { "item": "kitchen", "chance": 75, "x": 11,  "y": 11 },
+                { "item": "trash", "chance": 30, "x": 11,  "y": 19 },
+                { "item": "magazines", "chance": 25, "x": [ 3, 4 ],  "y": 17 },
+                { "item": "homebooks", "chance": 25, "x": [ 3, 4 ],  "y": 13 },
+                { "item": "bed", "chance": 80, "x": [ 18, 19 ],  "y": [ 7, 8 ] },
+                { "item": "kitchen", "chance": 75, "x": 11,  "y": [ 15, 18 ] },
+                { "item": "bedroom", "chance": 80, "x": [ 15, 19 ],  "y": [ 6, 12 ] },
+                { "item": "livingroom", "chance": 80, "x": [ 3, 7 ],  "y": [ 13, 19 ] },
+                { "item": "cleaning", "chance": 75, "x": [ 15, 16 ],  "y": [ 17, 18 ] },
+                { "item": "dresser", "chance": 70, "x": 14,  "y": [ 14, 15 ] },
+                { "item": "kitchen", "chance": 75, "x": [ 8, 10 ],  "y": [ 13, 19 ] },
+                { "monster": "GROUP_ZOMBIE", "chance": 2, "x": [ 2, 21 ],  "y": [ 2, 21 ] }
```

Comment:
This house is kinda profitable, I think: piles of stuff and low chance for zeds.  Gonna tweak this in the merge.